LondonBritishLibrary/orient/BLorient2082.xml

```diff
@@ -0,0 +1,375 @@
+                                        አበዊነ፡ አባ፡ ሚካኤል፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ አትሪብ፡ <sic>ወመሊዝ፡</sic> ወአባ፡ ዮሐንስ፡ ኤጲስ፡ ቆጶስ፡ ዘሀገረ፡ ብሩልስ፡ ወካልአን፡ አበው፡ መምህራነ፡ ቤተ፡
+                                        ክርስቲያን። ወውእቱ፡ ክፍል፡ ዳግማይ፡ እመጽሐፈ፡ ስንክሳር፡ ዘበትርጓሜሁ፡ ጉባኤ፡ ኵሎሙ፡ ገድላት፡ ዘሰማዕታት፡ ወጻድቃን። በረከተ፡ ጸሎቶሙ፡ የሀሉ፡ ምስለ፡ 
+                                        ፍቅርቶሙ፡ እኅተ፡ ክርስቶስ፡ ለዓለመ፡ ዓለም፡ አሜን። </incipit>
+                                </msItem>
+                                <msItem xml:id="p1_i1.2">
+                                    <locus from="1ra" to="24vb"/>
+                                    <title ref="LIT2375Synaxa#Maggabit"/>
+                                    <note>Three lines on <locus target="#23va"/> in the commemoration on 29 
+                                        <foreign xml:lang="gez">maggābit</foreign> have been erased.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.3">
+                                    <locus from="25ra" to="45vb"/>
+                                    <title ref="LIT2375Synaxa#Miyazya"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.4">
+                                    <locus from="46ra" to="71va"/>
+                                    <title ref="LIT2375Synaxa#Genbot"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.5">
+                                    <locus from="72ra" to="96vb"/>
+                                    <title ref="LIT2375Synaxa#Sane"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.6">
+                                    <locus from="97ra" to="130rb"/>
+                                    <title ref="LIT2375Synaxa#Hamle"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.7">
+                                    <locus from="131ra" to="154rb"/>
+                                    <title ref="LIT2375Synaxa#Nahase"/>
+                                </msItem>
+                                <msItem xml:id="p1_i1.8">
+                                    <locus from="155ra" to="159va"/>
+                                    <title ref="LIT2375Synaxa#Pagwemen"/>
+                                </msItem>
+                                <colophon xml:id="coloph1">
+                                    <locus target="#159vb"/>
+                                    <note>The note states that this manuscript was bought by 
+                                        <persName role="donor" ref="PRS14454EhataKrestos">ʾƎḫǝta Krǝstos</persName> from
+                                        <persName role="owner" ref="PRS14455AsqaGiyorgis">ʾAṣqa Giyorgis</persName> and offered to the 
+                                        <placeName ref="LOC000000000000000000000000000">church of St George</placeName> in the tenth year of the 
```

Comment:
You do not need to mark up everything if you are not creating IDs. just remove the tags

Reply:
Yes, I understand. I have only marked it in case you have a recommendation on how to identify this church or how to deal with it. If not, I will remove the mark.